### DISCUSSION TITLE
Resolved: Tabconf

### DISCUSSION THREAD

**Discussion by tcharding** [rust-bitcoin/rust-bitcoin#1650] [General]
I [submitted a workshop](https://github.com/TABConf/2023.tabconf.com/issues/34) for tabconf 2023. While writing the submission I had the idea that it would be cool if it was a Rust community workshop, led by a bunch of devs from our project and also from libraries up the stack (electrs, LDK, BDK etc.) - aiming at bringing us all into a room together to share ideas and help application devs use our libraries. And basically just get everyone psyched to hack Bitcoin in Rust.

If you are going to tabconf please holla at me and I'll add your name to the submission (assuming you want to join in with the workshop). I've never been to tabconf and don't know how it works re presenters and tickets so if you have a ticket for some other reason already let me know so I can keep in it mind when talking to the organisers.

If you are from one of the downstream projects using `rust-bitcoin` and you'd like to get involved please don't be shy!

If I've made any mistakes in the submission please tell me.

**Comment by apoelstra** [comment]
I will be there and I'm happy to show up for a rust-bitcoin workshop!

**Reply by tcharding** [reply]
Legend.

**Reply by sanket1729** [reply]
I plan to attend and can be a substitute for Andrew :)

**Reply by apoelstra** [reply]
For context, since somehow it has not appeared in this thread: even on Feb 20 when I posted "I'll be there" I shoud've known that I would not actually be there, since TABconf is scheduled to occur less than a week after my child is due to be born.

Since then, nothing has changed that trajectory :)

**Reply by yancyribbens** [reply]
there's a few threads about this going on https://github.com/rust-bitcoin/rust-bitcoin/issues/1868.  Seems like discussions is a better place to discuss than issues though :)

**Reply by apoelstra** [reply]
Quick update: the baby arrived a little early, on Aug 27, but not early enough that I will be able to make it to Tabconf :). Have fun without me!

**Comment by Kixunil** [comment]
Damn that's quite far away from me. I don't expect to come even though it looks great.

**Comment by yancyribbens** [comment]
It's also a long way for me to travel (and expensive) so I don't expect I'll make it this time around.  It looks great though.

**Comment by DanGould** [comment]
I will be at TABConf and happy to show up for the workshop 🤩

**Reply by tcharding** [reply]
Legendary!

**Comment by tcharding** [comment]
Woops I started a new discussion when we had one already, re-posting, sorry about the noise.

Hi y'all,

Anyone attending TABConf that would like to help out with the rust-bitcoin workshop I've written it up at: https://github.com/tcharding/workshop

There are three available tasks each with skeleton and a solution. Please come along, heckle, help, or hinder as amuses you.

Its on at 1pm-3pm on day 1.

Cheers.

**Comment by tcharding** [comment]
Stale, closing.
